contracts/plugins/assets/erc20/RewardableERC20.sol

```diff
@@ -24,6 +26,11 @@ abstract contract RewardableERC20 is IRewardable, ERC20 {
     mapping(address => uint256) public accumulatedRewards; // {qRewards}
     mapping(address => uint256) public claimedRewards; // {qRewards}
 
+    // Used to keep track of how many reward the Vault has accumulated
+    // Whenever _claimAndSyncRewards() is called we will calculate the difference
+    // between the current balance and `previousBalance` to figure out how much to distribute
+    uint256 internal previousBalance = 0;
```

Comment:
i think `lastRewardBalance` is a bit more explicit

Reply:
Good point